lnp2p/msg.go

```diff
@@ -7,7 +7,7 @@ import (
 
 // FIXME This is a stub function that just calls out to the lnutil lib for later.
 func processMessage(b []byte, peer *Peer) (lnutil.LitMsg, error) {
-	m, err := lnutil.LitMsgFromBytes(b, peer.GetIdx())
+	m, err := lnutil.LitMsgFromBytes(b, peer.Idx)
```

Comment:
This also makes things annoying to change later because if we change how Idxes are stored then we have tons of code reaching directly into the Peer datastructure to get that information instead of calling a function that we can change the implementation of more easily.

Reply:
`GetIdx()` just returned the Idx earlier. `peer.Idx` gets it directly. Again, using an accessor function makes sense _if_ you're doing something complex, but this function used to return `peer.Idx` anyway, hence the removal of that function. In the future, if we do something with peer.Idx that warrants it to be accessed in a different way than it is right now, we can use an accessor function then. Just that it doesn't have utility now :)

Reply:
The point is that if we have to change it to be something nontrivial then the only thing we have to change *is* the function.

Reply:
As I said, I don't think `peer.Idx` (or any field that is exported) will change because we have handlers higher up that do other stuff. Some fields are still unexplored because we expect them to change :)